io/disk/tiobench.py.data/tiobench.yaml

```diff
@@ -0,0 +1,26 @@
+setup: 
+    location: !mux
+        default:
+            target: '/mnt'
+
+    block: !mux
+        first:
+            blocks: 4096
+        second:
+            blocks: 8192
+
+    thread: !mux
+        default:
+            threads: 10
+
+    blocksize: !mux
+        default: 
+            size: 1024
+     
+    runs: !mux
+        default: 
+            numruns: 2
+  
+    iteration: !mux
+        default:
+            iterations: 5
```

Comment:
The `iterations` itself won't work. Either you have to list the iterations along with `!mux`:

```yaml
    iterations: !mux
        default: !mux
            1:
            2:
            3:
            4:
            5:
```

or you need to implement the iterations inside the test.

I'm actually wondering why do you need iterations, I'd just increase the `numruns` if I wanted multiple results. What is the benefit in executing the test multiple times?